php/mail/mail_utilities.php

```diff
@@ -1,28 +1,26 @@
 <?php
-function error($code, $txt) {
-    echo "Critical error " . $code . " - " . $txt;
-    return $code;
-}
+include_once '../db_connect.php';
+include_once '../general_utilities.php';
 
 function verifyNewAccount($id) {
     try {
         $sqlQuery = "UPDATE users SET is_verified = true WHERE id = :id AND is_verified = false";
-        $dbh = new PDO('pgsql:host=localhost;port=5432;dbname=matcha_db');
+        $dbh = connectToDatabase();
         $dbh->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
         $qres = $dbh->prepare($sqlQuery);
         $qres->bindParam(':id',$id);
         $qres->execute();
-        //$qres = $qres->fetchAll(PDO::FETCH_ASSOC);
     } catch (PDOException $msg) {
         error(5.03, $msg->getMessage());
         die();
     }
 }
 
 function getUserData($id) {
+
     try {
         $sqlQuery = "SELECT * FROM users WHERE id = :id";
-        $dbh = new PDO('pgsql:host=localhost;port=5432;dbname=matcha_db');
+        $dbh = connectToDatabase();
         $dbh->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
```

Comment:
Maybe, next time it is better to import from connection file stuff with already set attributes?

Reply:
I've thoroughly tested the other way of export  - a complete string with config. It is not OK for some reasons.
This way works fine so I'd better leave it 'as is' & move on